BetterPasswordPlugin.inc.php

```diff
@@ -208,17 +236,62 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+	/**
+	 * Create a temporary file with the data of password blacklists.
+	 * @return string temporary filename string
```

Comment:
This actually returns `string|boolean|null`, with `false` or `null` in the case of error.